### ISSUE TITLE
UTF-8 is a supported schema encoding in WMF5, Test-xDscSchema should be updated

### ISSUE THREAD

**Issue by TravisEz13** [PowerShell/xDSCResourceDesigner#35]

**Comment by bgelens** [comment]
Either this or New-xDscResource should produce schema.mof in ASCII format to comply with GH contribution tests

**Comment by TravisEz13** [comment]
I don't think this is an either or... The default should probably be Ascii, for backwards compatibility.  Do you want to file the issue?

**Comment by bgelens** [comment]
Opened an issue: https://github.com/PowerShell/xDSCResourceDesigner/issues/49
